godtools/Managers/FirstLaunchInitializer.swift

```diff
@@ -15,84 +15,85 @@ class FirstLaunchInitializer: GTDataManager {
     private let initialPackageCodes = ["kgp", "fourlaws", "sat"]
     
     func initializeAppState() {
-        let language = initializeInitialLanguage()
-        let resources = initializeInitialResources(language: language)
-        initializeInitialTranslations(language: language, resources: resources)
-        
-        saveToDisk()
+        try! realm.write {
```

Comment:
This will also make the app crash, right? Thats intentional or are you thinking to add a catch?